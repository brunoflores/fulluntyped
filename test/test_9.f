let x = ref 5 in x;
let x = ref 5 in !x;
