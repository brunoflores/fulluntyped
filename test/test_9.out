<loc #0> : Ref Nat
5
  : Nat
